Update standard ruleset text

amy-silcock opened the pull request:
Updating text to match newly created rulesets which are now more comprehensive.

@PetyaKangalova please can you check text changes?
@Ocre42 I've updated the link to the json ruleset at the end of the page. Please can you check that's correct?

amy-silcock committed:
Update standard ruleset text

Updating text to match newly created rulesets which are now more comprehensive.

andylolz commented on en/rulesets/standard-ruleset.rst:
```diff
@@ -1,78 +1,39 @@
-
-* Either recipient-country or recipient-region MUST be reported at activity level or transaction level but not both.
-
+* The recipient region must only be published in addition to the recipient country.
```
@amy-silcock Where does this rule come from?

I’m not aware of this rule. It’s not my understanding of the way `recipient-region` works.

rorads replied:
+1 this is contra my understanding of recipient regions & countries, so it would be good to have some consultation on it. I'd argue that it would create a lot of redundancy in hierarchical activity models (wherein there's a regional high level activity with multiple national activities underneath (some recent HUM publishers do this).

amy-silcock replied:
There are a few different rules and guidelines around reporting recipient countries and regions. The part we can't check is: 'Recipient-region **must** not be used merely to describe the region of a country reported in recipient-country, but ONLY if the region is a recipient IN ADDITION to the country.'

In my understanding this is to say if your activity benefits country x 100%, you must not also add the region it is within.

Perhaps it's my wording that's unclear?

andylolz replied:
Thanks @amy-silcock – that clarifies things. That’s my understanding of the rule in question, too, so I agree that there’s a problem with the wording.

> recipient region must only be published **in addition to** the recipient country

^^ This wording seems to suggest `recipient-region` can’t appear on its own, but only when accompanied by a `recipient-country`.

However, I think the main problem here is that this rule shouldn’t be included with the standard ruleset, because as you correctly say, it’s not possible to check using computer code. The standard ruleset is a collection of rules "[that a computer could, in principle, test.](http://reference.iatistandard.org/203/rulesets/standard-ruleset/#standard-ruleset)" This one can’t be, so it shouldn’t be included here.

amy-silcock replied:
Thanks @andylolz I'm proposing to include it in the 'non-machine readable rules' section for exactly that reason. These are rules which a computer can't (at least can't without additional information) check.

andylolz replied:
The name is confusing, but according to their definition, the "non-machine readable" rules **can** be checked by a computer.

The [standard ruleset page](http://reference.iatistandard.org/203/rulesets/standard-ruleset/) says:

> The Standard Ruleset is a collection of rules that cover some of the basic requirements of the IATI Standard and **that a computer could, in principle, test.**

Here’s what it says about non-machine readable rules:

> Some rules do not fit into this format, we call them “Non-Machine Readable Rules”. **It should be possible to write bespoke computer code to test each rule, but it requires a bit more work, both in terms of understanding what the rule means and the effort required to check.**

Also worth taking a look at #347. Hayden states that some rules “[…] cannot be 'easily' machine-checked, so would not fit within the [Standard Ruleset](http://iatistandard.org/202/rulesets/standard-ruleset/) as it is currently defined”. I think that’s also the case for this rule.

amy-silcock committed:
Updated region rule

Reporting region non-machine readable rule wording has been updated. It now matches the wording in the IATI Standard: http://reference.iatistandard.org/203/activity-standard/iati-activities/iati-activity/recipient-region/

andylolz commented:
@Ocre42 @amy-silcock I don’t think this PR should have been merged – I think it includes a number of problems.

Regarding [this discussion](https://github.com/IATI/IATI-Extra-Documentation/pull/535#discussion_r273021328), I think part of the confusion comes from the names “machine readable” and “non-machine readable”, which are a bit misleading given their definitions. Here’s another past comment that confirms the definition of non-machine readable rules: https://github.com/IATI/IATI-Extra-Documentation/issues/250#issuecomment-59192198

> The Non-Machine Readable rules could be tested by a computer, it's just we don't have them in a generic format that a generic ruleset parser can interpret. There should be no rules on that page which can not be tested (at least with bespoke code) by a computer. Your 'a title is meaningful' example is not on that page for this reason.

This PR also makes it less clear that `reporting-org/@ref` must match the org ID recorded in the IATI Registry metadata, because the rule is grouped under the heading `IATI Org IDs@ref`.

Additionally, this PR removes various newlines, meaning text renders in [very large paragraphs](https://github.com/IATI/IATI-Extra-Documentation/blob/063e46caeec22daaca083c0baf72149a470d6b51/en/rulesets/standard-ruleset.rst#standard-ruleset).

amy-silcock commented:
I've made some small adjustments: https://github.com/IATI/IATI-Extra-Documentation/pull/546
